earth_enterprise/src/server/mod_fdb/fusiondbservice.cpp

```diff
@@ -103,6 +103,8 @@ int FusionDbService::ProcessFusionDbRequest(
   bool is_time_machine = false;
 
   if (cmd_or_path == "dbRoot.v5") {
+    ap_log_rerror(APLOG_MARK, APLOG_DEBUG, 0, r,
```

Comment:
Are these statements meant to be permanent? Same question for all changes in this file and mod_fdb_impl.cpp.

Reply:
(I did delete over 100 other debug log statements before the commit, but chose to keep these 3)